serverless/discover/about-serverless.adoc

```diff
@@ -8,6 +8,13 @@ toc::[]
 
 {ServerlessProductName} provides Kubernetes native building blocks that enable developers to create and deploy serverless, event-driven applications on {product-title}. {ServerlessProductName} is based on the open source link:https://knative.dev/docs/[Knative project], which provides portability and consistency for hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
+[NOTE]
+====
+Because {ServerlessProductName} releases on a different cadence from {product-title}, the {ServerlessProductName} documentation does not maintain separate documentation sets for minor versions of the product. The current documentation set applies to all currently supported versions of {ServerlessProductName} unless version-specific limitations are called out in a particular topic or for a particular feature.
+
+For additional information about the {ServerlessProductName} life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift#ossrvless[Platform Life Cycle Policy].
```

Comment:
Checking whether just "Platform Life Cycle Policy" is intentional here, or if it was meant to be the full page title "OpenShift Container Platform Life Cycle Policy".

Edit: I see it's duplicated the same below as well, so I guess intentional!

Reply:
Yep, intentional, following the same format as in the SM docs.